Makefile

```diff
@@ -87,7 +87,7 @@ fmt:
 .PHONY: fmt-docs
 fmt-docs:
 	@echo ">> format markdown document"
-	$(MDOX) fmt --soft-wraps -l $$(find . -name '*.md' -not -path "./ui/node_modules/*" -not -path "./ui/prometheus-plugin/node_modules/*" -print) --links.validate.config-file=./.mdox.validate.yaml
+	$(MDOX) fmt --soft-wraps -l $$(find . -name '*.md' -not -path "./ui/node_modules/*" -not -path "./ui/prometheus-plugin/node_modules/*"  -not -path "./ui/storybook/node_modules/*"  -print) --links.validate.config-file=./.mdox.validate.yaml
```

Comment:
Ran into this when trying to format the markdown locally. Fixing while I'm in here!